cache.c

```diff
@@ -209,7 +209,7 @@ void cache_remove_ti(struct track_info *ti)
 static int read_cache(void)
 {
 	unsigned int size, offset = 0;
-	struct stat st = {};
+	struct stat st;
```

Comment:
That's odd because we're explicitly passing `-Wno-missing-field-initializers` to the compiler. So either way you should never see that warning.